dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/{{cookiecutter.project_name}}.Tests.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Alter the Teste project folder name 
from:
{{cookiecutter.project_name}}/test/{{cookiecutter.project_name}}.Tests/FunctionTest.cs
to:
{{cookiecutter.project_name}}/test/project.Tests/FunctionTest.cs